test/ILLink.RoslynAnalyzer.Tests/DynamicallyAccessedMembersAnalyzerTests.cs

```diff
@@ -678,6 +678,43 @@ private void M1()
 				.WithArguments ("type", "System.C.M2(System.Type)", "System.C.M1()", "'DynamicallyAccessedMemberTypes.PublicMethods'"));
 		}
 
+		[Fact]
+		public Task ConversionOperation ()
+		{
+			var TargetParameterWithAnnotations = @"
+namespace System
+{
+    class ConvertsToType
+    {
+        public static implicit operator Type(ConvertsToType value) => typeof (ConvertsToType);
```

Comment:
Is there a test for putting `[DAM]` on the return type of the implicit conversion method?

Reply:
It's similar to the case where the return type isn't annotated at all but I added this test to make sure it behaves as expected.